python/docs/_scripts/renderer.py

```diff
@@ -108,48 +108,52 @@ def _render_table(self, rows, headers):
     # render_annotation method --------------------------------------------------------
 
     @dispatch
-    def render_annotation(self, el: str) -> str:
+    def render_annotation(self, el: str) -> str:  # noqa: F811
```

Comment:
It may be possible to use:

https://beartype.github.io/plum/integration.html

The pattern described there to make this compatible with both `mypy` and `flake8`, if I am reading it correctly.

Reply:
I'll try this.  Great find!